packages/shared/lib/sdk/sync.ts

```diff
@@ -824,10 +824,10 @@ export class NangoAction {
                             Authorization: `Bearer ${this.nango.secretKey}`,
                             'Content-Type': 'application/json'
                         },
-                        data: {
+                        data: stringifyObject({
```

Comment:
I was not very confident about giving a string to axios but seems to work, would be great if you can try on your side just to confirm 😬

Reply:
It think it works because we explicitly set the content type to `application/json` and don't rely on Axios automatic content-type inference

Reply:
yep most probably